src/api.js

```diff
@@ -2,9 +2,15 @@ const fetchDefaults = {
   credentials: 'include', // required for Firefox 60, which is used in werkplekken
 };
 
-const apiCall = (url, opts) => {
+const apiCall = async (url, opts) => {
   const options = { ...fetchDefaults, ...opts };
-  return window.fetch(url, options);
+  const response = await window.fetch(url, options);
+  if (response.status === 403) {
+    const data = await response.json();
+    alert(data.detail);
+    window.location.href = '/';
```

Comment:
I don't think this is useful nor correct - it could be that the form start point is hosted on `/subpath/` and then this is wrong.

The basePath is passed as part of the `sdk.js` `OpenForms` constructor.

The better approach would be to still raise/throw the exception and handle it in the ErrorBoundary.

Example of ErrorBoundary usage: https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/Form.js#L162
Definition of ErrorBoundary: https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/ErrorBoundary.js
Documentation on error boundaries: https://reactjs.org/docs/error-boundaries.html

Reply:
Ah cool,  I didn't know React had this ErrorBoundary thing.  Definitely useful for this situation!